tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -535,7 +536,7 @@ def test_fsdp_load_old_checkpoint(
 
     if (composer_version in ['0.22.0', '0.23.0'] and version.parse(torch.__version__) < version.parse('2.3.0')) or (
         composer_version == '0.24.0' and version.parse(torch.__version__) < version.parse('2.4.0')
-    ) or (composer_version == '0.25.0' and version.parse(torch.__version__) < version.parse('2.5.0')):
+    ) or (composer_version in ['0.25.0', '0.26.0'] and version.parse(torch.__version__) < version.parse('2.5.0')):
```

Comment:
Were these checkpoints written out with a torch 2.4 image or torch 2.5 image? If torch 2.4, then the version check is wrong (should be `version.parse('2.4.0')` instead).

While checking this, would it be possible for you to verify the torch version that the composer 0.25.0 checkpoints were written with? They are not currently being tested by torch 2.4 tests, which I think is a mistake since we didn't even have torch 2.5 compatibility at the time 0.25.0 was released. @dakinggg since you did the release for that, is that right?

Reply:
Agreed. @irenedea  can we remove this entire clause checking if <2.5 and kick off an actions run to see what happens?

The 2.4 check was specifically because 2.4 broke something.

Reply:
they were written using the torch 2.5 image

Reply:
Ah nice, what about the 0.25.0 checkpoints

Reply:
not sure, @dakinggg ?

Reply:
i trust y'all, entirely possible I made a mistake last time

Reply:
ok yeah i removed this check completely and it failed daily tests. Is that what you meant @mvpatel2000 [b599d50](https://github.com/mosaicml/composer/pull/3683/commits/b599d50e9ba9ca6fced4e0931f3fc7d9a415a437)